services/web/server/tests/unit/with_dbs/06/test_projects_02.py

```diff
@@ -421,16 +420,17 @@ async def _open_project(
                 data, error = await assert_status(resp, e)
                 return data, error
             except AssertionError:
-                # re-raies if last item
+                # re-raise if last item
                 if e == expected[-1]:
                     raise
                 continue
     else:
-        return await assert_status(resp, expected)
+        data, error = await assert_status(resp, expected)
+        return data, error
 
 
 async def _close_project(
-    client, client_id: str, project: Dict, expected: web.HTTPException
+    client, client_id: str, project: Dict, expected: Type[web.HTTPException]
```

Comment:
when do we need to put that Type[] ?

Reply:
in this case yes, because ``expected`` is **a class** (not an instance) derived from ``web.HTTPException``